gmp/gmp.php

```diff
@@ -677,6 +677,57 @@ function gmp_rootrem (GMP $a, $nth) {}
  */
 function gmp_nextprime ($a) {}
 
+/**
+ * Calculates binomial coefficient
+ *
+ * @link https://www.php.net/manual/en/function.gmp-binomial.php
+ *
+ * @param resource|GMP|string|float|int $a
+ * @param int $b
+ * @return GMP|false
+ *
+ * @since 7.3
+ */
+function gmp_binomial($a, $b) {}
+
+/**
+ * Computes the Kronecker symbol
+ *
+ * @link https://www.php.net/manual/en/function.gmp-kronecker.php
+ *
+ * @param GMP|string|float|int $a
```

Comment:
Seems `resource` param type is missed here

Reply:
The PHP documentation is a little strange.

The function exists since 7.3 so I think the part "Either a GMP number resource in PHP 5.5 and earlier" should never apply.

Reply:
Oh, right. I see. But in this case I think that `resource` should be removed from `gmp_binomial` function as it completely vise-versa case. Function is available since 7.3 but resource parameter for php 5.5 and earlier.

Reply:
You're right, fixed.